modules/openapi-generator/src/main/resources/JavaSpring/converter.mustache

```diff
@@ -18,12 +18,12 @@ public class EnumConverterConfiguration {
 {{#models}}
 {{#model}}
 {{#isEnum}}
-    @Bean
-    Converter<{{{dataType}}}, {{name}}> {{classVarName}}Converter() {
-        return new Converter<{{{dataType}}}, {{name}}>() {
+    @Bean(name = "{{configPackage}}.EnumConverterConfiguration.{{classVarName}}Converter")
```

Comment:
This is changing how beans are defined by the generator.

I feel like this should be behind a configuration flag to allow generating with fully qualified bean names, or relying on the (current) simple `@Bean` annotation

Reply:
resolving, more discussion below